registry.json

```diff
@@ -304,7 +306,7 @@
   "theseus": "npm:theseus",
   "theseus-angular": "npm:theseus-angular",
   "topo": "github:capaj/topo-browser",
-  "timers": "github:jspm/nodelibs-timers",
+  "timers": "github:jspm/nodelibs-timers",c
```

Comment:
Thanks for pointing out @guybedford , I've removed the char, must have entered it by mistake.